src/ImageRunModal.jsx

```diff
@@ -1054,6 +1064,27 @@ export class ImageRunModal extends React.Component {
                                     onPlus={() => this.onPlusOne('healthcheck_retries')}
                                     onChange={ev => this.onValueChanged('healthcheck_retries', parseInt(ev.target.value) < 0 ? 0 : ev.target.value)} />
                         </FormGroup>
+                        {this.props.version.localeCompare("4.3", undefined, { numeric: true, sensitivity: 'base' }) >= 0 &&
```

Comment:
Locale specific version comparison sounds weird/dangerous. It seems better to split on `.` and compare them as lists, which seems to work very nicely. (e.g. `[4,2] > [4,3]`). But if `numeric` DTRT here, fine.

Reply:
Right, that would require fixing of.. multiple occurrences:

```
[jelle@t14s][~/projects/cockpit-podman]%git grep localeCompare | wc -l
6
```

I'd prefer to do this in a follow up.

Reply:
So, what about we also stop using passing version around everywhere because that is super annoying. Maybe we should use a context provider or useContext? Because we pass a *ton* of properties around.